bundles/com.salesforce.b2eclipse.jdt.ls/src/main/java/com/salesforce/b2eclipse/managers/BazelBuildSupport.java

```diff
@@ -148,6 +158,14 @@ public void deleteInvalidProjects(Collection<IPath> rootPaths, ArrayList<IProjec
             }
         }
     }
+    
+    public static void calculateExcludedFilePatterns(String bazelWorkspaceRootDirectoryPath) {
+        if (calculatedExcludedFilePatterns.isEmpty()) {
+            calculatedExcludedFilePatterns.addAll(EXCLUDED_FILE_PATTERN.stream()
```

Comment:
might be it is better to avoid unnecessary intermediate collection and adding the same item twice to the intermediate collection and then to the target? E.g. what if to redesign the code like it is given below? 
```
            EXCLUDED_FILE_PATTERN.stream()
            .map(path -> StringUtils.join("**" + bazelWorkspaceRootDirectoryPath, path))
            .forEachOrdered(calculatedExcludedFilePatterns::add);
```